neuralmonkey/tests/test_vocabulary.py

```diff
@@ -49,6 +49,7 @@ def test_there_and_back_self(self):
     def test_min_freq(self):
 
         vocabulary = Vocabulary()
+        vocabulary.correct_counts = True
```

Comment:
no ja si myslim, ze to prepinatko by melo byt automaticky ... proc by se mel s tim nekdo otravovat vzdyv nejhorsi varianta je ze si to bude myslet ze ma vsechno frekvenci 1. Skoro bych to i cele zahodil a udelal bez nej kde frekvence ukladal vzdycky bez ohledu jestli jsou platne